blog/2023-11-06-idp-data-preso/index.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: optimizing-data-quality-in-dev-portals
+title: Optimizing for Data Quality in your Developer Portal
+authors: [ghaynes, ksmith, bromano, nabuskey, jsanford, nimak]
+tags: [backstage, dev portal, data ingestion]
+---
+*Disclaimer: CNOE makes strong assumptions about using a subset of open source technologies when building Internal Developer Platforms (IDPs). Recommendations made and patterns discussed are hence centered around the exact tooling that CNOE adopts to implement a capability within an IDP. In this particular post, we assume Kubernetes as the orchestrator platform, Backstage as the technology that implements the developer portal capability, Argo CD for continuous delivery, and Crossplane or Terraform for infrastructure as code (IaC). Capability and technology names may be used interchangeably in the blog post but practices discussed are primarily around the specific set of technologies named earlier.*
+
+## Establishing the source of truth
+
+There are different approaches to representing entities like Kubernetes objects and cloud resources in Backstage. In such context, *platform engineers need to optimize for creation of reliable data*. The last thing you as a platform provider want to see happen is to lose trust of end users because you are displaying incorrect information. There are however, a number of key decisions to be made when building entity representations in Backstage. Particularly:
+
+1. What gets represented in Backstage and what doesnt
+2. How to ensure the Backstage entity offers an accurate representation of its real world counterpart.
```

Comment:
```suggestion
2. How to ensure the Backstage entity offers an accurate representation of its real world counterpart
```

I'm not an expert, but I know a personal nit pick of mine is when lists have inconsistencies in punctuation usage 😅